.github/workflows/e2e.yml

```diff
@@ -0,0 +1,34 @@
+name: E2E Tests
+
+on:
+  pull_request:
+    branches: [ master, release/** ]
+  push:
+    branches: [ master, release/** ]
+    tags: [ v* ]
+
+jobs:
+  Run-E2E-Tests-1:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # checkout current branch
+      - name: Run E2E tests
+        uses: ./.github/actions/run-e2e
+        with:
+          packages: 'contract.opcodes, contract.openzeppelin'
+  Run-E2E-Tests-2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # checkout current branch
+      - name: Run E2E tests
+        uses: ./.github/actions/run-e2e
+        with:
+          packages: 'contract.precompile, contract.records'
+  Run-E2E-Tests-3:
```

Comment:
While a decent solution to use a reusable action, it was probably more overall lines of code than using a simpler job matrix for the different packages. Just FYI for future.